coverage/env.py

```diff
@@ -148,7 +145,7 @@ class PYBEHAVIOR:
     soft_keywords = (PYVERSION >= (3, 10))
 
     # PEP669 Low Impact Monitoring: https://peps.python.org/pep-0669/
-    pep669 = bool(getattr(sys, "monitoring", None))
+    pep669: Final[bool] = bool(getattr(sys, "monitoring", None))
```

Comment:
Conceptually, all of PYBEHAVIOR should be Final.  How come only this attribute needed Final?

Reply:
That's a good question. I think it has to do with the file exclusions I mentioned above. Probably they would all require `Final` if everything were included.